apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -308,7 +309,7 @@ func (m *ModelManagerAPI) createModelNew(
 	// Create the model information in the model database. This information
 	// is read-only and is used for providers and brokers without the need
 	// to query the controller database.
-	if err := modelInfoService.CreateModel(ctx, m.controllerUUID); err != nil {
+	if err := modelInfoService.CreateModel(ctx, m.controllerUUID, version.Current); err != nil {
```

Comment:
Should we just let the service default this value off as business logic? We don't let people specify the agent version when creating a model and it makes the facade have to understand this.